app/controllers/v1/frameworks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class V1::FrameworksController < APIController
+  skip_before_action :authenticate, :reject_without_user!
+
+  def index
+    frameworks = Framework.all.sort_by(&:name)
```

Comment:
How about using ActiveRecord's native sorting (i.e. `Framework.order(:name).all`) ?

Reply:
@leeky does it make more sense to order by number?